operator/pkg/validate/validate_values_test.go

```diff
@@ -35,6 +35,9 @@ func init() {
 }
 
 func TestValidateValues(t *testing.T) {
+	// Maistra skips these tests because it adds TLS configuration to the charts
+	// but doesn't add it to the corresponding Istio types.
+	t.Skip("https://github.com/istio/istio/issues/1000000")
```

Comment:
Keeping tests skipped based on the discussion in the previous PR https://github.com/maistra/istio/pull/562#issuecomment-1321712774